### ISSUE TITLE
Neutral buoyancy

### ISSUE THREAD

**Issue by thereporter42** [StrangeLoopGames/EcoIssues#9010]
Version: 0.7.6.3 beta

Steps to Reproduce:
1. Find a deep section of water in the game
2. Go out far enough that you can go deep down
3. Press the left control key to dive down
4. Try and pick up a trap or rock or anything else on the ocean bottom

Expected behavior:
Should stay where you are when you let go of the ctrl key. 

Actual behavior:

When you let go of the control key you immediately start to float up towards
the surface. 

This makes it very hard if you are trying to mine rock off the bottom of the ocean or 
pick up a trap or anything else under water.

Really What should happen is unless you press the spacebar to surface you
should stay where you are. 

This will really make it a lot better and easier for the players when doing 
under water activities such as mining under water or tending to 
fishing traps etc. 

Right now if you let go of the ctrl key you immediately start to float up
towards the surface and it makes it that much harder to do anything under
water.  Trying to hold the ctrl key and hit another key at the same time. 

In reality divers would have weights on to keep them submersed.  Can you please
look at fixing this please. 

https://www.scubadiving.com/top-5-ways-to-master-buoyancy-control-for-scuba-divers

**Comment by Conchipedia** [comment]
i agree the Swimming does need a little tweak, i make Doros that come out in to water and come out always with the ability to stay where i am mid water, But to get lower i have to Hit space before i can hit CTRL to make myself sink, this does get a bit annoying after awhile as you go shooting up in the water before you can get yourself back down again, working underwater is not to bad once you get used to the way of holding CTRL key while working as this does force you to use Calories while working underwater

**Comment by thereporter42** [comment]
It is a more of a "quality of life" thing than just a suggestion it would make 
life a lot easier in game when playing this game if we had neutral buoyancy 

It would make things a lot easier to do under water.

**Comment by Conchipedia** [comment]
I am hoping that Underwater Building/Farming Etc gets looked at in more detail for later releases
